clms-mobile/app/AppNavigator.tsx

```
@@ -0,0 +1,40 @@
+import {router, Stack, useRouter} from 'expo-router';
+import axios from "axios";
+import { useAuth } from '@/core/contexts/AuthenticationContext';
+
+const AppNavigator = () => {
+    const { accessToken, isAuthenticated } = useAuth();
+    const router = useRouter();
+
+    useEffect(() => {
+        // Clear axios authorization header if not authenticated
+        if (!isAuthenticated) {
+            delete axios.defaults.headers.common['Authorization'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)